docs/src/index.md

```diff
@@ -1,91 +1,75 @@
 # KernelAbstractions
 
-`KernelAbstractions.jl` is a package that allows you to write GPU-like kernels that
-target different execution backends. It is intended to be a minimal, and performant
-library that explores ways to best write heterogenous code.
+[`KernelAbstractions.jl`](https://github.com/JuliaGPU/KernelAbstractions.jl) (KA) is
+a package that allows you to write GPU-like kernels targetting different
+execution backends. KA intends to be a minimal and
+performant
+library that explores ways to write heterogeneous code. Although parts of
+the package are still experimental, it has been used successfully as part of the
+Exascale Computing Project to run Julia code on pre-Frontier and pre-Aurora
+systems. Currently missing from the documentation is
+ * how to debug kernels, and
+ * how to profile kernels.
```

Comment:
At least for CUDA GPU's, you can use this: https://cuda.juliagpu.org/dev/development/profiling/

Reply:
Reformulated. How about this?